src/SFA.DAS.LevyTransferMatching.Web/Models/Cache/RejectApplicationCacheItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Cache
+{
+    public class RejectApplicationCacheItem
```

Comment:
This doesn't appear to be used

Reply:
This has been removed now. I have created another branch that will probably be required for the browser to remember the selected application when the users will jump between the pages. But that's another Task/Story for later.

Reply:
This class still seems to exist, but I don't think it has a purpose?

Reply:
Yes, true, it's not been used. I have removed it now.